src/escape_string.cpp

```diff
@@ -321,15 +321,24 @@ namespace libtorrent
 		return ret;
 	}
 
-	std::string base32encode(std::string const& s)
+	std::string base32encode(std::string const& s, int flags)
 	{
-		static const char base32_table[] =
+		static const char base32_table_canonical[] =
 		{
 			'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H',
 			'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P',
 			'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X',
 			'Y', 'Z', '2', '3', '4', '5', '6', '7'
 		};
+		static const char base32_table_lowercase[] =
```

Comment:
would you mind adding a test exercising this code path as well? see test/test_string.cpp